src/main/resources/templates/message.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Message Detail</title>
+</head>
+<body>
+<form action="/logout" method="post" style="display: inline">
+    <button type="submit">Logout</button>
+</form>
+<a href="/"><button>Home</button></a>{{currentUser}} is currently logged in
+<hr>
+<p>{{message.content}}</p>
+<p>Created by <a href="/user?id={{message.user.id}}">{{message.user.username}}</a> on {{message.createdOn}}</p>
+{{#access}}
```

Comment:
As discussed this does seem hokey but mustache is all about making their templates logic-less so the controller must be in charge of it.